kurl_proxy/cmd/main.go

```diff
@@ -78,6 +78,17 @@ func main() {
 	}
 	secrets := clientset.CoreV1().Secrets(namespace)
 
+	_, err = secrets.Get(context.Background(), tlsSecretName, metav1.GetOptions{})
+	if err != nil {
+		log.Print("tls secret not found")
```

Comment:
if everything works, this will look like an error even if it isn't.  how about "creating tls secret"